service.js

```diff
@@ -8,98 +8,134 @@ var restify = require('restify');
 var request = require('request-promise');
 const bunyan = require("bunyan");
 
-class HealthCheckService{
+class HealthCheckService {
     constructor(services) {
         //let's check the list list of services if OK:
-        if(services && Array.isArray(services)) {
+        if (services && Array.isArray(services)) {
             services.forEach((service) => {
-                if (typeof service.name != 'string' || typeof service.url != 'string'){
+                if (typeof service.name != 'string' || typeof service.url != 'string') {
                     throw "the list of monitored services is malformed";
                 }
             });
         }
-        this.services=services;    
+        this.services = services;
         //default logger
         this.logger = bunyan.createLogger({
             name: 'health-check-default',
             streams: [
                 {
-                  level: 'info',
-                  stream: process.stdout
+                    level: 'info',
```

Comment:
Not sure why I cannot change this level of logging. 
Even if I change the LOG_LEVEL environment variable

Reply:
There was a problem with the environment variable case: loglevel instead of logLevel...